ui/src/onboarding/actions/index.ts

```diff
@@ -1,10 +1,12 @@
+// Libraries
+import _ from 'lodash'
+
 // Constants
 import {StepStatus} from 'src/clockface/constants/wizard'
 import {SetupSuccess, SetupError} from 'src/shared/copy/notifications'
 
 // Actions
 import {notify} from 'src/shared/actions/notifications'
-
 import {client} from 'src/utils/api'
```

Comment:
nit- maybe this should go up under `//Libraries` ?

Reply:
I added it under // APIs :)